resources/api-gateway/templates/webhook-certificates.yaml

```diff
@@ -8,12 +8,17 @@ metadata:
   annotations:
     sidecar.istio.io/inject: "false"
 spec:
-  replicas: 1
   template:
     metadata:
       annotations:
         sidecar.istio.io/inject: "false"
     spec:
+      securityContext:
+        runAsUser: 65534
+        runAsGroup: 65534
+        runAsNonRoot: true
```

Comment:
Shall we also include `readOnlyRootFilesystem: true`?

Reply:
Thanks, good finding! It's not relevant for Pod Security Admission, but still we should have it.